### ISSUE TITLE
very awesome :)

### ISSUE THREAD

**Issue by gedw99** [choria-io/asyncjobs#38]
Just wanted to say how great this looks. Thank you for all the hard work on this @ripienaar 

I am going to try it out and feedback if thats ok ?

Love how the CLI wraps the NATS CLI to give an easy way to interact with the system.

**Comment by ripienaar** [comment]
Thanks for the kind words, happy to have feedback :)

**Comment by ripienaar** [comment]
But do read the `Status` section :) its super early days

**Comment by gedw99** [comment]
yep i know it's all a work in progress :) thanks for the warning though.

Am not going to use this in Prod... 

Part of the reason i was so excited to see this is because , a higher order Task system on top on NATS is a low risk proposition because all the really tricky stuff is already worked out t the NATS level.

**Comment by ripienaar** [comment]
Yeah indeed, though not to say its not difficult, I want to support things like composite tasks where a task is a group of tasks with dependencies.  This is really hard in this model, but I have some ideas :)
